aws/sdk/sdk-external-types.toml

```diff
@@ -1,21 +1,30 @@
 # These are the allowed external types in the `aws-sdk-*` generated crates, checked by CI.
 allowed_external_types = [
-    "aws_credential_types::*",
-    "aws_http::*",
-    "aws_runtime::*",
-    "aws_smithy_async::*",
-    "aws_smithy_http::*",
-    "aws_smithy_runtime::*",
-    "aws_smithy_runtime_api::*",
     "aws_smithy_types::*",
+    "aws_credential_types::*",
     "aws_types::*",
-
-    "http::header::map::HeaderMap",
-    "http::header::value::HeaderValue",
+    "aws_smithy_runtime_api::*",
+    "aws_smithy_runtime::*",
+    "aws_smithy_async::rt::sleep::*",
+    "aws_smithy_async::time::*",
+    "aws_smithy_async::future::pagination_stream::PaginationStream",
+    # only allowed in from impl for presigned request
     "http::request::Request",
-    "http::error::Error",
-    "http::uri::Uri",
-    "http::method::Method",
+
+    ### All below this line need to be fixed:
```

Comment:
The event stream one at the very bottom doesn't need to be fixed. It just needs a tooling upgrade.

Reply:
the crate won't go 1.0 though—we might want a newtype?